src/OSPSuite.Assets/UIConstants.cs

```diff
@@ -553,6 +553,18 @@ public static string WithParent(int uniqueIndex)
          }
       }
 
+      public static class ValueOrigins
```

Comment:
Those are all default stuff that we can change once we have the latest and greatest documentation. All icons are also dummy one at the moment